README.md

```diff
@@ -42,5 +42,31 @@ queue.add_item(value) # adding item
 
 list_of_values = queue.get_items(10) # getting items
 queue.ack_items(list_of_values) # acknowledging items or
-queue.revert_items(list_of_values) # reverting item to queue
+queue.revert_items(list_of_values) # reverting items to the queue
+```
+
+###Pool###
+Use `from pyrq import Pool`.
+
+If you need slave synchronization, use *synced_slaves_count* and *synced_slaves_timeout* arguments.
+
+This data structure serves for managing queues of any **stringable** items. Typical data flow consist of several phases:
```

Comment:
**serializable** by bylo přesnější, pojem _stringable_ moc neexistuje.

Btw tenhle odstavec spíš odpovídá obecnému fungování front, takže platí i pro `Queue`. Ale u `Queue` nemáš žádný odstavec, kterýho popisuje, je tam jen původní příklad.  To README pak vypadá poněkud nekonzistentně